.github/workflows/sync_docs.yml

```diff
@@ -66,7 +69,6 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v7
         with:
-          token: ${{secrets.GH_ACCESS_TOKEN}}
           path: gitbook-docs
           commit-message: "Update docs from ${{ env.REPO_NAME }} (commit: ${{ github.sha }})"
           committer: "devops@taluslabs.xyz <devops@taluslabs.xyz>"
```

Comment:
So this would be:

* GH actions bot to create the PR
* devops-at-talus to sign the commits
(if I'm right)